cpp/core/degibbs/degibbs.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2008-2025 the MRtrix3 contributors.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Covered Software is provided under this License on an "as is"
+ * basis, without warranty of any kind, either expressed, implied, or
+ * statutory, including, without limitation, warranties that the
+ * Covered Software is free of defects, merchantable, fit for a
+ * particular purpose or non-infringing.
+ * See the Mozilla Public License v. 2.0 for more details.
+ *
+ * For more details, see http://www.mrtrix.org/.
+ */
+
+#pragma once
+
+#include <complex>
+
+namespace MR::Degibbs {
+
+typedef double real_type;
```

Comment:
Suggest using the more modern version:
```
using real_type = double;
using complex_type = std::complex<real_type>;
```

Reply:
Closed by 7dfe81b5289fc5b8676909a6f1e80912cf31a293.